modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -48,18 +48,22 @@ def submit_form_2122 # rubocop:disable Metrics/MethodLength
             unless power_of_attorney.persisted?
               power_of_attorney = ClaimsApi::PowerOfAttorney.find_by(md5: power_of_attorney.md5)
             end
+
+            if feature_enabled_and_claimant_present?
+              update_auth_headers_for_dependent(
+                power_of_attorney,
+                claimant_information
+              )
+            end
+
             power_of_attorney.auth_headers['participant_id'] = target_veteran.participant_id
+            power_of_attorney.auth_headers['file_number'] = file_number
             power_of_attorney.save!
```

Comment:
since we are not passing this directly to the job in this controller anymore I am adding it to the headers to utilize later on.

Open to ideas on how to adjust for this differently.

Reply:
why not just pass the `dependent_claimant_poa_assignment_service` object through the jobs? 🤷

![dependent-assignment](https://github.com/user-attachments/assets/64e26b29-306a-4b55-bc16-df14a7efa398)

Reply:
Because it goes through that job and into the assignment job as a string.  I was unable to figure out how to stop sidekiq from turning that object into a string, but. I did try that and also dug into if I could undo that behavior.  Passing it to the first job from the controller worked as expected, passing it from the first sidekiq job into the second sent that object as a string, even if I created the service in that first job instead.

If you know how to undo that totally open to reverting this change, but I was unable to find a way to handle that

Reply:
We also use those header values later on for naming, so since we cannot pass the object through sidekiq (would some type of method to format it after sending I suspect) and since we need the headers either way, that is what I went with, that being said, obviously totally open to whatever ends up being cleaner.

Reply:
Oh, right. Sidekiq saves the arguments as JSON in Sidekiq, so it will `.to_s` the object on subsequent calls.

Other than temporarily storing these values to the database, I think auth_headers are our best bet.